python/tests/viz/extensions/test_reports.py

```diff
@@ -18,3 +23,13 @@ def test_exception_if_not_both_profiles(self, profile_view):
         with pytest.raises(ValueError, match="This method has to get both target and reference profiles"):
             report = SummaryDriftReport(ref_view=profile_view, target_view=None)
             report.report()
+
+    def test_summary_profile_returns_html(self, profile_summary_report):
+        html_report = profile_summary_report.report()
+        assert isinstance(html_report, HTML)
+        assert "<div>" in html_report.data
+
+    def test_exception_if_not_profile(self, profile_view):
```

Comment:
what if you have defined only reference_view? how does it behave?

Reply:
actually, it expects only `target_view` as argument. In this case, is it relevant to add a case passing an unexpected argument?

Reply:
saw you've added a test for it =)